mesa_geo/geospace.py

```diff
@@ -1,45 +1,85 @@
+from __future__ import annotations
+from typing import List, Optional, Union, Tuple
```

Comment:
Not a blocker to this PR getting merged, but just wondering if we can do `for __future__ import union_operator` or something to do `A | B` instead of `Union[A, B]`.